### PR TITLE
Pre-release fixes and corrections

### PR DESCRIPTION
**Pull request by juangallostra**
## Description of the issue/feature this PR addresses

Prepare `senaite.sync` for its first release.  

## Current behavior before PR

`senaite.sync`, although already functional, needed some final touches before being ready to be released. Some of these are: Copyright notice in code files, Fixes in the README, ...

## Desired behavior after PR is merged

* All `.py` files have the proper copyright notice.
* Repo contains an updated `CONTRIBUTORS.rst` file.
* Repo contains scripts to update code headers and contributors.
* Mistakes in `README.rst` are fixed.

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: templates/write_code_headers.py.in [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+#
+# Scans a source tree for python files and writes the header template on top of each file
+
+import os
+import sys
+import optparse
+from datetime import datetime
+
+SRC_DIR = '${buildout:directory}/src/senaite/sync'
```
Comment (juangallostra) [inline templates/write_code_headers.py.in:10]: @xispa @ramonski I wasn't able here to find out how to specify the source directory so I finally run the script by passing the route as a command line argument. Any ideas on how it should be specified?

File: templates/write_contributors.py.in [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+#
+# Generates CONTRIBUTORS.rst file, with the name and email of the contributors
+# to the project, sorted from more to less commits.
+
+import os
+import re
+import subprocess
+import sys
+
+SRC = os.getcwd()
```
Comment (juangallostra) [inline templates/write_contributors.py.in]: @xispa @ramonski same here as my previous comment. For some reason `${buildout:directory}` didn't get much love.

File: templates/write_code_headers.py.in (hunk 1 above)
Comment (juangallostra) [reply templates/write_code_headers.py.in:10]: Ok, got it!

Comment (ramonski) [reply templates/write_code_headers.py.in:10]: Hi @juangallostra, can you please hook this script in the local `buidlout.cfg` like here: https://github.com/senaite/senaite.core/blob/master/buildout.cfg#L60

File: CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+This file is autogenerated each time a release is published.
+
+Contributions to master, excluding merge commits, sorted by number of
+contributions descending.
+
+- Nihad Mammadli, 1992.nihad@gmail.com
+- Juan Gallostra, jgallostra@naralabs.com
+- Ramon Bartl, rb@ridingbytes.com
+- Jordi Puiggené, jpuiggene@naralabs.com
```
Comment (ramonski) [inline CONTRIBUTORS.rst:12]: I should do more commits 😄

**Pull request merged**